app/views/posts/index.html.erb

```diff
@@ -1 +1,27 @@
-<h1>Here is a list of all posts for a user</h1>
\ No newline at end of file
+<div class="post-page">
+  <div class="user-cont">
+    <div>
+      <img src="https://user-images.githubusercontent.com/37116322/222183427-3450696b-429a-4e9a-9e19-48bbe4b48edf.png" alt="profile">
+    </div>
+    <div class="user-info">
+      <h2><%= @user.name %></h2>
+      <div>
+        <p>Number of posts: <%= @user.posts_counter %></p>
+      </div>
+    </div>
+  </div>
+  
+  <% @posts.each do |post| %>
+    <div class="post-cont">
+      <h3>Post #<%= post.id %></h3>
+      <p><%= post.text %></p>
+      <span>Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></span>
+      <%= link_to 'View', "/users/#{@user.id}/posts/#{post.id}" %>
+    </div>
+    <div class="comment-cont">
+      <% post.last_five_comments.each do |comment| %>
+        <p><%= comment.user.name %>: <%= comment.text %></p>
+      <% end %>
+    </div>
+  <% end %>
```

Comment:
## [OPTIONAL]
- [ ] You can also create a partial for @posts and render them just like I have suggested to you for the users.